persistence/rest/src/main/java/org/infinispan/persistence/rest/configuration/RemoteServerConfiguration.java

```diff
@@ -0,0 +1,73 @@
+package org.infinispan.persistence.rest.configuration;
+
+import org.infinispan.commons.configuration.ConfigurationInfo;
+import org.infinispan.commons.configuration.attributes.Attribute;
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.commons.configuration.elements.DefaultElementDefinition;
+import org.infinispan.commons.configuration.elements.ElementDefinition;
+
+/**
+ * @since 10.0
+ */
+public class RemoteServerConfiguration implements ConfigurationInfo {
+
+   static final AttributeDefinition<String> HOST = AttributeDefinition.builder("host", null, String.class).immutable().autoPersist(false).build();
+   static final AttributeDefinition<Integer> PORT = AttributeDefinition.builder("port", 80).immutable().autoPersist(false).build();
```

Comment:
I would have preferred a single attribute in the form host:port instead of an extra element here as well, but I guess here it's not as clear cut as with the counter bounds.

Reply:
https://issues.jboss.org/browse/ISPN-10545